astropy/coordinates/solar_system.py

```diff
@@ -504,12 +501,17 @@ def get_moon(time, location=None, ephemeris=None):
 
     Notes
     -----
+    {_EPHEMERIS_NOTE}
     """
 
     return get_body('moon', time, location=location, ephemeris=ephemeris)
 
 
-get_moon.__doc__ += indent(_EPHEMERIS_NOTE)[4:]
+# Add note about the ephemeris choices to the docstrings of relevant functions.
+# Note: sadly, one cannot use f-strings for docstrings, so we format explicitly.
+for f in [f for f in locals().values() if callable(f) and f.__doc__ is not None
+          and '{_EPHEMERIS_NOTE}' in f.__doc__]:
```

Comment:
I've also long wished python would allow for the comprehension syntax

```python
for f in locals().values() if (...):
   ...
```

Reply:
Here, the main reason I did the list comprehension there is that I cannot just iterate over `locals.values()` anyway (I guess since I define `f`). But yet, the comprehension syntax there would make things just a little more compact and less indented.